.env.example

```diff
@@ -2,23 +2,22 @@ API_CLIENT_ID=approved-premises-ui
 APPROVED_PREMISES_API_URL=http://localhost:8080
 CAS1_E2E_ADMINISTRATOR_PASSWORD=secret
 CAS1_E2E_ADMINISTRATOR_USERNAME=JIMSNOWLDAP
-CAS1_E2E_ASSESSOR_EMAIL=jim.snow@example.digital.justice.gov.uk
-CAS1_E2E_ASSESSOR_NAME="JIM SNOW"
+CAS1_E2E_ASSESSOR_EMAIL=jim.snow@justice.gov.uk
+CAS1_E2E_ASSESSOR_NAME="Test Jim Snow"
```

Comment:
is it worth sync this with CAS2/3?

Reply:
I had to update these locally to get local E2E passing for CAS1, so as long as those equivalent tests are passing on CAS2/3 it should be fine?